src/android/CameraPreviewFragment.java

```diff
@@ -119,6 +148,49 @@ public void startCamera() {
         }
     }
 
+    public Size calculateResolution(int targetSize) {
+        int orientation = getResources().getConfiguration().orientation;
+        Size calculatedSize;
+        if (orientation == 1) {
```

Comment:
Instead of checking against `1`, check against `Configuration.ORIENTATION_PORTRAIT` for readability. [SO1](https://stackoverflow.com/questions/48777084/android-activity-getresources-getconfiguration-orientation-returning-wrong) and [SO2](https://stackoverflow.com/questions/2795833/check-orientation-on-android-phone) also mention problems on some devices when using this method. Worth testing locked orientation?